### ISSUE TITLE
Text Align Label Axis Bullet

### ISSUE THREAD

**Issue by jeinhorn787** [amcharts/amcharts5#838]
**Question**
Hello!

Is it possible to vertical align axis bullet label items in a horizontal bar chart?

I am trying to text align the `Label` being used as the sprite for `AxisBullet` on the Y axis.  I am targeting the numbers stacked vertically on the right side of the chart in the pictures below.  They are `Label` elements being used as the `sprite` passed to the `AxisBullet`.  The text is vertically left aligned but I want it vertically right aligned.  I tried putting `textAlign: "right"` on the `Label` element but that does not work.  Any help is appreciated!

**Source code**
```
    const root = am5.Root.new(id);

    const chart = root.container.children.push(
      am5xy.XYChart.new(root, {
        panX: false,
        panY: false,
      })
    );

    const yRenderer = am5xy.AxisRendererY.new(root, {
      inversed: true,
    });
    yRenderer.labels.template.set("visible", false);
    yRenderer.grid.template.set("visible", false);

    const yAxis = chart.yAxes.push(
      am5xy.CategoryAxis.new(root, {
        renderer: yRenderer,
        categoryField: "name",
        bullet: (root, axis, dataItem) => {
          const { value } = dataItem.dataContext;

          const label = am5.Label.new(root, {
            text: value,
            centerX: am5.p0,
            centerY: am5.p50,
            textAlign: "right" // does not work
          });

          label.adapters.add("x", () => {
            return chart.plotContainer.getPrivate("width") - 150;
          });

          return am5xy.AxisBullet.new(root, {
            location: 0.5,
            sprite: label,
          });
        },
      })
    );

    yAxis.data.setAll(data);

...
```

**Current**
![image](https://user-images.githubusercontent.com/29580247/223220165-a1b11d6b-8701-47a0-a840-6525556d56a8.png)

**What I Want**
![image](https://user-images.githubusercontent.com/29580247/223172407-a22ec43c-e233-47ee-a6f1-85373762e2d8.png)

**Comment by zeroin** [comment]
Have you tried setting `centerY: am5.p100` for the Label?
If this won't help, it would be great if you could create a working codepen which we could edit - thanks.

**Comment by jeinhorn787** [comment]
I found the solution!

So I was doing some additional logic in the `x` adapter to take away some width from the `getPrivate("width")` call.  I thought I needed that at first as the labels were being blown way out to the right but turns out that was part of the problem. The other part was that I did not define a `width` on the label but it seems as if that is needed here. Though it is strange because both `100%` and `0` for width work.

Here is the code pen, look for comments on lines 66 and 70 for instructions on how to view the solution.

https://codepen.io/jeinhorn787/pen/xxaXOQe
